src/main/java/com/footbalmanager/app/abstraction/BaseController.java

```diff
@@ -0,0 +1,9 @@
+package com.footbalmanager.app.abstraction;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@CrossOrigin("http://localhost:8080")
+public abstract class BaseController {
```

Comment:
Hej mogłeś zrobić z tego własną adnotacje:

@CrossOrigin("http://localhost:8080")
@RestController
public @interface RestController {
}

i nie musiałbyś wszędzie extendować basecontrollera

Reply:
Sprawdzwe rano czy zadziala :) samo zadeklarowanie interfejsu nie dzialalo bo nie widzial controllerow :) zobacze z @Public bo nie bardzo wiek co to :p

Reply:
Sprawdzwe rano czy zadziala :) samo zadeklarowanie interfejsu nie dzialalo bo nie widzial controllerow :) zobacze z @Public bo nie bardzo wiek co to :p